Word_Break_139.py

```diff
@@ -0,0 +1,25 @@
+# Time - O(N ^ 2)
```

Comment:
As discussed, substring operation takes O(n) time. So total time complexity is O(n^3)